docs/developer/guidelines/dashboards.md

```diff
@@ -1,3 +1,158 @@
 # Dashboards
 
 -----
+
+Datadog dashboards enable you to efficiently monitor your infrastructure and integrations
+ by displaying and tracking key metrics.
+ 
+## Integration Preset Dashboards
+
+If you would like to create a default dashboard for an integration, follow the guidelines in the [Best Practices](./dashboards.md#best-practices) section.
+
+### Exporting a dashboard payload
+When you've created a dashboard, you can export the dashboard payload to be included in the integration assets.
+
+Run the following command to [export the dashboard](../ddev/cli.md#export):
+
+```cli
+ddev meta dash export <URL_OF_DASHBOARD> <INTEGRATION>
```

Comment:
Maybe we should tell they need to configure the keys before being able to do that (https://datadoghq.dev/integrations-core/ddev/cli/#set)